### PR TITLE
Feature/event taxa images

### PR DESCRIPTION
**Pull request by jack-brinkman**
Merging work for #229

### CODE REVIEW

File: packages/graphql-api/src/resources/shared/resources/taxonMedia/taxonMedia.type.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  extend type Query {
+    taxonMedia(key: String, size: Int, from: Int): [Image!]!
+  }
+
+  type Image {
+    identifier: String!
+    type: String!
+    subtypeLiteral: String
+    title: String
+    MetadataDate: String
```
Comment (MortenHofft) [inline packages/graphql-api/src/resources/shared/resources/taxonMedia/taxonMedia.type.js:13]: What is up with the casing here in this type?

File: packages/react-components/locales/translations/en/components/images.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "identifier": "ID",
+  "type": "Type",
+  "subtypeLiteral": "Subtype",
+  "title": "Title",
+  "rights": "Rights",
+  "Credit": "Credit",
```
Comment (MortenHofft) [inline packages/react-components/locales/translations/en/components/images.json:7]: again - I do not think casing should change.

File: packages/react-components/locales/translations/en/components/images.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "providerLiteral": "Provider",
+  "description": "Description",
+  "accessURI": "URL",
+  "accessOriginalURI": "Original URL",
+  "format": "Format",
+  "PixelXDimension": "Width",
+  "PixelYDimension": "Height"
```
Comment (djtfmartin) [inline packages/react-components/locales/translations/en/components/images.json:14]: casing issues here as well

**Comment by jack-brinkman** [comment]
@djtfmartin @MortenHofft I kept the casing consistent with the casing in the [Audubon Core Term List](https://ac.tdwg.org/termlist/#ac_MetadataDate) - happy to change the casing if you'd prefer it to be consistent.

**Comment by djtfmartin** [comment]
@jack-brinkman ah - i see - thats awkward : )
I'd probably still lower case them for consistency. What do you think @MortenHofft  ?

**Comment by MortenHofft** [comment]
It is probably most convinient for usage and reading to have them aligned, but also a good argument to keep the consistent with the standard

I guess it depends on wether you think the APIs should mirror the standards or just have a mapping to it. The GBIF APIs follow tdwg mostly, but not strictly it seems to me. We change casing for basisOfRecord values for example.

I asked @timrobertson100 who suggested to keep the casing. I can personally live with both versions - at least I know understand the reasoning for the surprising casing.

**Comment by jack-brinkman** [comment]
Okay - that sounds good to me - casing consistency across the API makes sense. I'll change the casing 👍

**Comment by jack-brinkman** [comment]
I've made a few tweaks to the ALA `taxonMedia` source to make it consistent/reliable with the `size` & `from` parameters.

@MortenHofft @djtfmartin happy to make any further PR changes you see fit before merging

**Comment by jack-brinkman** [comment]
Closing to squash and re-merge

**Comment by MortenHofft** [comment]
Thanks for changing that - Feel free to merge

**Pull request merged**